README.md

```diff
@@ -2,35 +2,40 @@
 
 Uses the Brother Print SDK for Android...
 
-More info can be found here, including a list of compatible printers: http://www.brother.com/product/dev/mobile/android/index.htm
+More info can be found here, including a list of compatible printers: [http://www.brother.com/product/dev/mobile/android/index.htm](http://www.brother.com/product/dev/mobile/android/index.htm)
 
 Already bundled is the following version: v3.0.4 (5/18/2016) which is in the `src/android/libs` dir. By downloading this you agree to the Brother SDK License terms which are included in the README under the libs dir.
```

Comment:
What version is bundled for IOS? 

Also, do we know if v3.0.4 is the current bundled version for Android? I'm guessing that's out of date...

Reply:
I did see that the `Android` `SDK` was updated not too long ago... but the `iOS` `SDK` hasn't been changed since May I think?

The current `SDK` version for `iOS` is `3.1.1`

The current `SDK` version for `Android` is `3.0.7`

I have not updated these binaries though.  At least, I don't think I did.